src/Spectre.libHeatmapDataScaling/HistogramEqualization.h

```diff
@@ -20,6 +20,9 @@ limitations under the License.
 
 namespace spectre::visualization
 {
+
+constexpr uint8_t intensitiesRange = 255;
```

Comment:
I am not sure we want to pollute the namespace `spectre::visualization` with a value so *specific* for this algorithm. I'd rather put this close to the place it is used: `HistogramEqualization.cpp`, probably at the top of the file.

I also think you should use [`std::numeric_limits<uint8_t>::max()`](http://en.cppreference.com/w/cpp/types/numeric_limits/max) instead.